vendor/github.com/openshift/cluster-api-actuator-pkg/pkg/e2e/framework/machines.go

```diff
@@ -14,18 +14,11 @@ import (
 	"k8s.io/client-go/rest"
```

Comment:
Is there a counter PR for this on `cluster-api-actuator-pkg`?

Reply:
https://github.com/openshift/cluster-api-actuator-pkg/pull/22